user_input/input.py

```diff
@@ -11,20 +11,14 @@
 ===================================================================
 """
 def response_option(max_len):
-    valid_response = False
-    while not valid_response:
-        response = str(input("Enter in a response with no more than {} words: ".format(max_len)))
-        if len(response.split()) > max_len:
```

Comment:
we are just going to truncate the response in `pad_sequence()` if more than `max_len` is inputted